torch/onnx/symbolic_opset9.py

```diff
@@ -1911,3 +1911,10 @@ def multinomial(g, input, num_samples, replacement=False, generator=None):
     return g.op("Multinomial", log_input,
                 dtype_i=sym_help.cast_pytorch_to_onnx['Long'],
                 sample_size_i=num_samples)
+
+
+def gelu(g, self):
+    _sqrt2 = 1.4142135623730951
+    erf = g.op('Erf', div(g, self, g.op('Constant', value_t=_sqrt2)))
```

Comment:
```suggestion
    erf = g.op('Erf', div(g, self, g.op('Constant', value_t=torch.Tensor([_sqrt2]))))
```

Also the CI is failing

> Aug 21 06:40:09 test/onnx/test_operators.py::TestOperators::test_gelu FAILED             [  2%]